cmd/parse_locals.go

```diff
@@ -113,20 +113,20 @@ func parseLocals(path string, terragruntOptions *options.TerragruntOptions, incl
 	}
 
 	// Decode just the Base blocks. See the function docs for DecodeBaseBlocks for more info on what base blocks are.
-	localsAsCty, trackInclude, err := config.DecodeBaseBlocks(terragruntOptions, parser, file, path, includeFromChild, nil)
+	createExtensions, err := config.DecodeBaseBlocks(terragruntOptions, parser, file, path, includeFromChild, nil)
```

Comment:
What is `createExtensions`? I don't see this in https://github.com/gruntwork-io/terragrunt/blob/main/config/config.go#L853

Reply:
You're looking at `main`. This is upgrading to `v0.54.13`. https://github.com/gruntwork-io/terragrunt/blob/v0.54.13/config/config.go#L853

I think I meant to say `contextExtensions` to keep in line with what that version says. Fixed.